csvqb/csvqb/models/cube/qb/components/measure.py

```diff
@@ -75,6 +75,20 @@ def new_measures_from_data(data: PandasDataTypes) -> "QbMultiMeasureDimension":
             [NewQbMeasure(m) for m in sorted(set(columnar_data))]
         )
 
+    @staticmethod
+    def existing_measures_from_data(
+        data: PandasDataTypes, csvw_column_name: str, csv_column_uri_template: str
+    ) -> "QbMultiMeasureDimension":
+        columnar_data = pandas_input_to_columnar_str(data)
+        return QbMultiMeasureDimension(
+            [
+                ExistingQbMeasure(
+                    uritemplate.expand(csv_column_uri_template, {csvw_column_name: m})
+                )
+                for m in sorted(set(columnar_data))
```

Comment:
Does this need to be sorted?

Reply:
Strictly, no, however I prefer idempotent outputs from processes like these. So some form of consistent ordering is necessary.

Reply:
Fine. Leave it.